### ISSUE TITLE
Broken waypoints on Knee Deep

### ISSUE THREAD

**Issue by whatcorey** [ineedbots/t4m_bot_warfare#2]
Seems the waypoints are completely broken for this map, bots stay in one spot, or barely move at all.

**Comment by megvadulthangya** [comment]
> Seems the waypoints are completely broken for this map, bots stay in one spot, or barely move at all.

You can use my version, of botwarfare I replaced the working wayponts.
[https://et.sytes.net/codwaw/mods/mp_awe5/z_svr_bots.iwd](https://et.sytes.net/codwaw/mods/mp_awe5/z_svr_bots.iwd)

**Comment by Splormf** [comment]
I've also had some problems getting the bots to work. I tried megvadulthangya files, but replacing them seemed to do nothing. I've tried editing the waypoints myself, but when I open the editor there are no waypoints, I can make a .csv, but I can't edit any of them. I can't figure it out. I've noticed that the bots on Banzai also wont traverse the bridge, which is a bummer because it would be a great camping spot for them. What directory is the game looking into for waypoints? perhaps i've installed it weird?

**Comment by megvadulthangya** [comment]
Bad upload... Try now i fixed

**Comment by Splormf** [comment]
I'm not sure if im doing something wrong. I replaced the .CSV files here \SteamLibrary\steamapps\common\Call of Duty World at War\Mods\mp_bots\waypoints, and I also replaced my z_svr_bots.iwd here: \SteamLibrary\steamapps\common\Call of Duty World at War\mp_bots with the one you uploaded megvadulthangya. The bots in knee deep still get stuck in the buildings. Thank you for helping though megvadulthangya.
